corehq/util/test_utils.py

```diff
@@ -912,12 +912,7 @@ def new_db_connection(alias=DEFAULT_DB_ALIAS):
     Use to test transaction isolation when a transaction is in progress
     on the current/existing connection.
     """
-    from django import VERSION as django_version
-    if django_version[:2] >= (4, 1):
-        connections.configure_settings({})
-    else:
-        connections.ensure_defaults(alias)
-        connections.prepare_test_settings(alias)
+    connections.configure_settings({})
```

Comment:
I should have noted this in the original PR where `connections.configure_settings()` was originally added. It's surprising to me that this does the same thing as the Django 3.x version since `alias` is not being passed in, and this somehow `prepare_test_settings` sounded better for this context. Can you point to the Django docs or explain how you decided this is equivalent to what it was doing before?

Reply:
Good question. I had linked to [this django change](https://github.com/django/django/commit/13378ad95206a96c3d1d261eb738032468a7acc1) in the PR where I [added support for 4.2](https://github.com/dimagi/commcare-hq/pull/34417). So the notable change is how we get the `conn` object to set defaults and test settings on. I don't recall why I chose to pass in an empty dict into `configure_settings`. However looking over it, it should fall back to whatever our `DATABASES` value is in settings, and based on the `testsettings.py` file, that should be [set](https://github.com/dimagi/commcare-hq/blob/master/testsettings.py#L157-L157) properly?